game_patch/multi/server.cpp

```diff
@@ -305,6 +316,37 @@ static void send_private_message_with_stats(rf::Player* player)
     send_chat_line_packet(str.c_str(), player);
 }
 
+void shuffle_level_array()
+{
+    std::ranges::shuffle(rf::netgame.levels, rng);
+    xlog::warn("Shuffled level rotation");
+}
+
+const char* get_rand_level_filename()
+{
+    std::size_t num_levels = rf::netgame.levels.size();
+
+    // current_level_index keeps our position in rotation while we go somewhere else
+    rf::String& filename = rf::level_filename_to_load;
+
+    if (num_levels <= 1) {
+        // nowhere else to go, we're staying here!
+        return filename.c_str();
+    }
+
+    std::uniform_int_distribution<std::size_t> dist_levels(0, num_levels - 1);
+    std::size_t rand_level_index = dist_levels(rng);
```

Comment:
Imo prefer `dist`.

Reply:
I disagree, I'd prefer to use uniquely named variables for `dist` in each instance `rng` is used (currently only this once, but eventually more).